templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>You want delete {{ car }}</h1>
+    <h1>Are you sure to delete this car?</h1>
```

Comment:
The sentence 'Are you sure to delete this car?' is slightly awkward. Consider changing it to 'Are you sure you want to delete this car?' for better clarity.